test/suite/wttest.py

```diff
@@ -582,6 +594,14 @@ def tearDown(self, dueToRetry=False):
                     else:
                         teardown_msg += "; " + str(tmp[1])
 
+        if self.__module__.startswith("test_layered"):
+            # FIXME-WT-15786: For tests using a follower config, we cannot rely on the getDisaggParameters
```

Comment:
I think the actual issue is that this field is only being used/updated when the disagg hook is in use. However, we don't use that hook with layered tests.